src/metrics/PrometheusRenderer.cpp

```diff
@@ -38,12 +38,12 @@ const std::string PrometheusRenderer::RenderMetrics(Server* server) {
     const std::string& metricType = std::get<1>(metric);
     const long long& metricValue   = std::get<2>(metric);
 
-    // Output the type of each metric
-    ss << "# TYPE " << metricName << " " << metricType << "\n";
-
     // Add prefix to metric key if set in configuration.
     const std::string metricKey = !config.get<std::string>("prometheus_metric_prefix").empty() ? (config.get<std::string>("prometheus_metric_prefix") + "_" + metricName) : metricName;
```

Comment:
actually `prometheus_metric_prefix` cannot be empty because it has a default so maybe this code should be simplified @olesku?

Reply:
I agree, we can drop the conditional here :)

Reply:
done